libs/common/src/lib/localization/localization.service.ts

```diff
@@ -96,4 +106,112 @@ export class LocalizationService {
     // Translate the text
     return t(text, params);
   }
+
+  async pullTranslations() {
+    const tx = createNativeInstance({
+      token: this.configService.get("TRANSIFEX_TOKEN"),
+      secret: this.configService.get("TRANSIFEX_SECRET")
+    });
+    const locales = ["fr_FR", "es_MX", "pt_BR"];
+    for (const locale of locales) {
+      const dbLocale = locale.split("_").join("-");
+      await tx.fetchTranslations(locale);
+      const translations = await tx.cache.getTranslations(locale);
+      const keys = Object.keys(translations);
+      for (const key of keys) {
+        const translation = translations[key];
+        const i18nItems = await I18nItem.findAll({ where: { hash: key } });
```

Comment:
I think you should be able to do this for all keys in `keys` in one query instead of doing it N+1 style in a loop.